warehouse/query-core/src/main/java/datawave/query/config/ShardQueryConfiguration.java

```diff
@@ -1648,6 +1650,14 @@ public void setDateIndexTimeTravel(boolean dateIndexTimeTravel) {
         this.dateIndexTimeTravel = dateIndexTimeTravel;
     }
 
+    public boolean getIgnoreExistentFields() {
+        return ignoreNonExistentFields;
+    }
+
+    public void setIgnoreExistentFields(boolean ignoreNonExistentFields) {
```

Comment:
It looks like you forgot the "Non" in this method name and the one above.

Reply:
ah, good catch. I'll fix this and recommit